src/main/java/org/cactoos/list/Sorted.java

```diff
@@ -66,7 +66,7 @@ public Sorted(final T... src) {
      */
     @SuppressWarnings("unchecked")
     public Sorted(final Iterable<T> src) {
-        this((Comparator<T>) Comparator.naturalOrder(), new ListOf<>(src));
+        this((Comparator<T>) Comparator.naturalOrder(), src);
```

Comment:
@victornoel Cactoos follows the *No instanceof, type casting, or reflection* principle. Thus, remove `(Comparator<T>)` cast.

Reply:
@fabriciofx please raise a bug for this, this is out of scope of my work here. Also see the comment above, this was done on purpose, I don't know why